modules/lnd.nix

```diff
@@ -23,11 +22,13 @@ let
     tor.streamisolation=true
 
     [Bitcoind]
-    bitcoind.rpcuser=${cfg.bitcoin-rpcuser}
-    bitcoind.zmqpubrawblock=ipc://${bitcoindDataDir}/zmqpubrawblock.sock
-    bitcoind.zmqpubrawtx=ipc://${bitcoindDataDir}/zmqpubrawtx.sock
+    bitcoind.rpcuser=${config.services.bitcoind.rpcuser}
+    bitcoind.zmqpubrawblock=tcp://127.0.0.1:28332
+    bitcoind.zmqpubrawtx=tcp://127.0.0.1:28333
```

Comment:
Nit, feel free to ignore. I think ideally we'd add `zmqpubrawblock` and `zmqpubrawtx` options to the bitcoin module with type `nullOr String` and default `null`. Then we can use them here with toString. As a result if you forget to set any of those options, the lnd module will throw errors at build time.

Reply:
makes sense, will add this as it's easy to do